Velaptor/Services/IDotnetService.cs

```diff
@@ -0,0 +1,27 @@
+namespace Velaptor.Services;
+
+/// <summary>
+/// Invokes Dotnet functions.
+/// </summary>
+public interface IDotnetService
```

Comment:
Please refactor this interface to be `internal`.

> [!Important]
> All new files should be internal by default.  This is to make sure that we don't expose types to the public API
> unless it is fully intended to be part of the public API.
> Another tip,  All new classes whether they are `internal` or `public` should by default be `sealed` unless there is a 
> reason to not have it `sealed`.  

It is always better to start the public API with a minimal surface of only what is needed and then open it up over time if there is a good reason to do so.  This way, we don't have to worry about nearly as many breaking changes to users.  Also, we get a small perf boost with `sealed` classes. 

Nice bonus!!